source/rules/literals.rs

```diff
@@ -239,7 +239,7 @@ fn string_single_quoted(input: &[u8]) -> Result<&[u8], Literal> {
         return Result::Error(Error::Code(ErrorKind::Custom(StringError::TooShort as u32)));
     }
 
-    if input[0] == 'b' as u8 {
+    if input[0] == 'b' as u8 || input[0] == 'B' as u8 {
```

Comment:
Whoops! I just thought of chaining the check with the parser (binary_check -> string|nowdoc), but I didn't notice those were different. Sorry!